routes/matches/matches.js

```diff
@@ -460,14 +469,28 @@ router.get("/:match_id", async (req, res, next) => {
       req.user !== undefined &&
       (matchRow[0].user_id == req.user.id || Utils.superAdminCheck(req.user))
     ) {
-      sql = "SELECT * FROM `match` where id=?";
+      //sql = "SELECT * FROM `match` where id=?";
+      sql =
+        "SELECT mtch.id, mtch.user_id, mtch.server_id, mtch.team1_id, mtch.team2_id, mtch.winner, " +
+        "mtch.team1_score, mtch.team2_score, mtch.team1_series_score, mtch.team2_series_score, " +
+        "mtch.cancelled, mtch.forfeit, mtch.start_time, mtch.end_time, " +
+        "mtch.max_maps, mtch.title, mtch.skip_veto, mtch.private_match, mtch.enforce_teams, mtch.min_player_ready, " +
+        "mtch.season_id, mtch.is_pug, mtch.map_sides, " +
+        "gs.ip_string, gs.port, gs.gotv_port, gs.display_name, mtch.team1_string, mtch.team2_string " +
+        "FROM `match` mtch " +
+        "LEFT JOIN game_server gs ON gs.id = mtch.server_id " +
+        "where mtch.id = ?";
     } else {
       sql =
-        "SELECT id, user_id, server_id, team1_id, team2_id, winner, " +
-        "team1_score, team2_score, team1_series_score, team2_series_score, " +
-        "team1_string, team2_string, cancelled, forfeit, start_time, end_time, " +
-        "max_maps, title, skip_veto, private_match, enforce_teams, min_player_ready, " +
-        "season_id, is_pug, map_sides FROM `match` where id = ?";
+        "SELECT mtch.id, mtch.user_id, mtch.server_id, mtch.team1_id, mtch.team2_id, mtch.winner, " +
+        "mtch.team1_score, mtch.team2_score, mtch.team1_series_score, mtch.team2_series_score, " +
+        "mtch.cancelled, mtch.forfeit, mtch.start_time, mtch.end_time, " +
+        "mtch.max_maps, mtch.title, mtch.skip_veto, mtch.private_match, mtch.enforce_teams, mtch.min_player_ready, " +
+        "mtch.season_id, mtch.is_pug, mtch.map_sides, " +
+        "gs.ip_string, gs.port, gs.gotv_port, gs.display_name, mtch.team1_string, mtch.team2_string " +
```

Comment:
Also again, unprotected route, please ensure the user is logged in and has proper access to this match. If not, remove the IP.